server.port=8001
spring.datasource.url=jdbc:mysql://mysql5045.site4now.net:3306/db_a69d44_lverde?allowPublicKeyRetrieval=true&useSSL=false
spring.datasource.username=a69d44_lverde
spring.datasource.password=Lverde21##@
spring.jpa.generate-ddl=true
spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver

lima.verde.client.url=localhost:4200
#Jpa/Hibernate :
spring.jpa.show-sql=false
spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect
spring.jpa.hibernate.ddl-auto = update
lima.verde.pagination.size = 10

springdoc.api-docs.path=/CanjearPuntos/v1/swagger

lima.verde.exception.default.message=Ocurrió un error interno.